sbrb-backend/app/routers/listing_route.py

```diff
@@ -33,10 +32,17 @@ def get_listing_by_id(
     user_id: int = Query(None, description="User ID"),
     db: Session = Depends(get_db),
 ):
+    if listing_id == 0:
```

Comment:
Looks like this condition is already handled in line 39.

Reply:
yup, you're right. i've removed the redundant check and the associated test case